core/kazoo_proper/src/pqc_cb_accounts.erl

```diff
@@ -66,6 +66,17 @@ allow_number_additions(AccountId) ->
                                           ,[{kzd_accounts:path_allow_number_additions(), 'true'}]
                                           ).
 
+-spec patch_account(pqc_cb_api:state(), kz_json:object(), kz_term:ne_binary()) -> pqc_cb_api:response().
+patch_account(API, ReqJObj, AccountId) ->
```

Comment:
I think `AccountId` should be second and `ReqJObj` should be third, similar to `topup_request`.